auto-reconfiguration.sh

```diff
@@ -9,6 +9,7 @@ VERSION=$(cat auto-reconfiguration-archives/version)
 INDEX_PATH="/auto-reconfiguration/index.yml"
 UPLOAD_PATH="/auto-reconfiguration/auto-reconfiguration-$VERSION.jar"
 
+rm -f auto-reconfiguration-archives/auto-reconfiguration-*sources.jar
```

Comment:
Ah! Right, I forgot I was modifying upstream code here. I *think* what is happening is their build process is using a resource that doesn't store the sources jar in the same directory, but I wasn't able to find it. 

This script will need a change, and there's two options I can see:

1) `rm` the sources jar
2) Loop over each jar in the dir and upload it. Or change the transfer to s3 script to accept multiple arguments.